src/web_xmlhttprequest.ml

```diff
@@ -86,10 +86,10 @@ let getAllResponseHeadersAsList x =
   | Ok s -> Ok
     ( s
       |> Js.String.split "\r\n"
-      |> Array.map (Js.String.splitAtMost ": " ~limit:2)
-      |> Array.to_list
-      |> List.filter (fun a -> Array.length a == 2)
-      |> List.map
+      |. Belt.Array.map (Js.String.splitAtMost ": " ~limit:2)
```

Comment:
What is with all these `|.` and where is the definition of it defined?

Reply:
`|.` should work in 2.2.3

Reply:
How do you inspect it's type?  I'm trying to type-hole it but it just says it doesn't exist...